### ISSUE TITLE
Systemd for create_bridges

### ISSUE THREAD

**Issue by henn** [CCI-MOC/hil#555]
Similar to #525 done by @SahilTikale , we need to make the call to `create_bridges` into a separate systemd script, as it only needs to run once per boot.

This will also entail updating [INSTALL.rst](https://github.com/CCI-MOC/haas/blob/27a013bc8bc954ffabf64389b3c097cc55ae894b/INSTALL.rst#networking---bridges).

Once this is done, users will no longer need to modify /etc/rc.local!!!

**Comment by SahilTikale** [comment]
Also, we need to use 'IP' commands to create vlan and stop using vconfig/net-tools because they are deprecated. 

--Sahil

-----Original Message-----

**Comment by starbops** [comment]
So I think #552 is no longer needed, right? Shall I close the PR?

**Comment by SahilTikale** [comment]
This issue becomes redundant once we have the openVPN feature.
Should we close it then @henn ?

**Comment by henn** [comment]
Sure

**Comment by naved001** [comment]
@shwsun We can close this issue. It's a duplicate of #671 which has been fixed by #674
